lib/Notes/NotePopupModal/NotePopupModal.js

```diff
@@ -157,11 +178,8 @@ const NotePopupModal = ({
       size="small"
       footer={footer}
     >
-      <div
-        className="ql-editor"
-        data-test-note-popup-modal-content
-        dangerouslySetInnerHTML={popupContentHTML}
-      />
+      { titleContent }
+      { popupContentHTML.__html && detailsContent }
```

Comment:
Pleasem keep code in one style
```suggestion
      {titleContent}
      {popupContentHTML.__html && detailsContent}
```